apps/condo/domains/billing/schema/resolvers/accountResolver.js

```diff
@@ -0,0 +1,97 @@
+// ELS is a 10-digit code, where 3 and 4 characters are russian letters and the rest are numbers.
+const IS_ELS_REGEXP = /^\d{2}[А-Я]{2}\d{6}$/i
+
+const { isEmpty, isNil } = require('lodash')
+
+const { generateGqlQueries } = require('@open-condo/codegen/generate.gql')
+const { generateServerUtils } = require('@open-condo/codegen/generate.server.utils')
+const { find } = require('@open-condo/keystone/schema')
+
+const { BILLING_ACCOUNT_OWNER_TYPE_COMPANY, BILLING_ACCOUNT_OWNER_TYPE_PERSON } = require('@condo/domains/billing/constants/constants')
+const { ERRORS } = require('@condo/domains/billing/constants/registerBillingReceiptService')
+const { clearAccountNumber, isPerson } = require('@condo/domains/billing/schema/resolvers/lib')
+const { Resolver } = require('@condo/domains/billing/schema/resolvers/resolver')
+
+const BILLING_ACCOUNT_FIELDS = '{ id  }'
+const BillingAccountGQL = generateGqlQueries('BillingAccount', BILLING_ACCOUNT_FIELDS)
+const BillingAccountApi = generateServerUtils(BillingAccountGQL)
+
+class AccountResolver extends Resolver {
```

Comment:
you should avoid adding any new method to the existing Resolver! create function rather than add new methods